utilities/write_batch_with_index/write_batch_with_index_internal.h

```diff
@@ -297,14 +296,18 @@ class WriteBatchWithIndexInternal {
   static const Comparator* GetUserComparator(const WriteBatchWithIndex& wbwi,
                                              uint32_t cf_id);
 
-  // For GetFromBatchAndDB or similar
-  explicit WriteBatchWithIndexInternal(DB* db,
-                                       ColumnFamilyHandle* column_family);
-  // For GetFromBatchAndDB or similar
-  explicit WriteBatchWithIndexInternal(ColumnFamilyHandle* column_family);
-  // For GetFromBatch or similar
-  explicit WriteBatchWithIndexInternal(const DBOptions* db_options,
-                                       ColumnFamilyHandle* column_family);
+  static const ImmutableOptions& GetImmutableOptions(
```

Comment:
Just wondering if there could be a better place for this (maybe in db_options.h?)

Reply:
I guess it could be a free-standing helper declared in `column_family.h`, where similar methods like `GetColumnFamilyID` live. Let me move it